amoni/api.py

```diff
@@ -230,7 +284,7 @@ def add_table(
     server_permissions: str = "full",
     columns: List = None,
 ):
-    columns = columns if columns is None else []
+    columns = columns if columns is not None else []
```

Comment:
Good catch! But I tend to prefer specifying the action for the True case first.

```suggestion
    columns = columns if [] if columns is None else columns
```